easistrain/EDD/preStraind0cstEDD.py

```diff
@@ -0,0 +1,235 @@
+
+import numpy as np
+import h5py
+import scipy.constants
+
+
+# fileRead = '/home/esrf/slim/easistrain/easistrain/EDD/Results_ihme10_glob_TiC.h5'
+# fileSave = '/home/esrf/slim/easistrain/easistrain/EDD/Results_ihme10_globstrain_TiC.h5'
+# pathFileDetectorCalibration = '/home/esrf/slim/easistrain/easistrain/EDD/Results_ihme10_align.h5'
+# scanDetectorCalibration = 'fit_0001_2_1'
+# pathFileAngleCalibration = '/home/esrf/slim/easistrain/easistrain/EDD/Results_ihme10_TiC_calib.h5' 
+# scanAngleCalibration = 'fit_0001_3'
+# numberOfPeaks = 5
+# d0 = [2.491844,2.158000,1.525936,1.301323,1.245922]
+
+
+def uChEConversion(a, b, c, ch, ua, ub, uc, uch):
+    return np.sqrt(
```

Comment:
For reference, `math.sqrt` is faster than `np.sqrt` when dealing with simple numbers (same for `cos`/`sin`...)